README.md

```diff
@@ -1,4 +1,8 @@
-# eslint-config-kintone [![npm version](https://badge.fury.io/js/eslint-config-kintone.svg)](https://badge.fury.io/js/eslint-config-kintone)
+# eslint-config-kintone
+
+[![npm version](https://badge.fury.io/js/eslint-config-kintone.svg)](https://badge.fury.io/js/eslint-config-kintone)
+[![CircleCI](https://circleci.com/gh/kintone/eslint-config-kintone.svg?style=svg)](https://circleci.com/gh/kintone/eslint-config-kintone)
```

Comment:
`?style=shield` is more suitable than `svg`

Reply:
Thanks! I've fixed it.